test/unit/app/utils/test_kenya_state_machine.py

```diff
@@ -56,6 +59,7 @@
      (initial_pin_entry_state, standard_user, "0000", "initial_pin_confirmation"),
      (initial_pin_entry_state, standard_user, "AAAA", "exit_invalid_pin"),
      # initial_pin_confirmation transitions tests
+     #TODO: remove these tests and the other testing of pin, they're redundant
```

Comment:
These tests don't test the pin alone, they also test that a specific transition works, it just happens to use the pin feature. In a sense they are still unit-tests  in that they test that that transition will behave in a specific way provided certain values. What do you think ? I think we should retain the so that every transition is tested, so that should the transition break for a reason other than the pin we can still catch trace the breakage.